encoding/preparation.md

```diff
@@ -296,4 +296,6 @@ Just extract it somewhere.
 
 ---
 
-[^1]: It should be noted that the author strongly disagrees with this sentiment. The two have a lot in common, and any capable Avisynth encoder could reach a similar leven in Vapoursynth within a few months, maybe even weeks. At least I'm honest, okay?
+[^1]: It should be noted that the author strongly disagrees with this sentiment.
```

Comment:
As noted in https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/commit/a0280ae57b29b1db294a51c06f4eec6d2b394c42, semantic linefeeds don't work for footnotes.